src/styles/stars.css

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+}
+
+.stars :last-child {
+  margin-right: 0;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: no-repeat url(../images/star.svg);
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```

Comment:
This rule duplicates '.stars :last-child' on 5-7 lines

Reply:
+1